### PR TITLE
module: generic: SUT_generic: Make evironment variables accessible

### PR DESCRIPTION
**Pull request by jkoeppeler**
Add the "environ" variables also to the setup and teardown script of the "sut_generic" sut-class.

### CODE REVIEW

File: module/generic/SUT_generic.py [hunk 1]
```diff
@@ -38,3 +38,7 @@ def _start(self):
 
     def stop(self):
         self.run_teardown_script()
+
+    def run_script(self, script):
+        if Path(script).is_file():
+            subprocess.call(str(script), shell=True, env=self.env)
```
Comment (nemethf) [inline module/generic/SUT_generic.py]: The implementation of the base-class sets `check=True` as a `subprocess.run` argument.  I don't remember why.  Maybe tipsy can treat the whole measurmenet as a failure if the script exists with an error code. At any rate, would it make sense to set `check=True` here as well?

Also, can you, please, add yourself to the end of the AUTHORS file in the root directory.

Thanks

Comment (jkoeppeler) [reply module/generic/SUT_generic.py]: Thanks, I replaces the `call` with `run` in all generic related code and added the `check=True`. My question would be if it makes sense to have environment variables in for all types, not only generic. Could be useful for the setup and teardown scripts.

Comment (nemethf) [reply module/generic/SUT_generic.py]: Originally, every configuration setting was defined in a json schema, so I don't think adding environment   variables there makes sense.  But later we wanted to reuse tipsy for measuring something else than a telco pipeline, so we quickly put together the generic modules.    For `SUT_generic` and `Tester_generic`, it is okay to set `env` for every `subprocess.run` call if that is useful for you.

Comment (jkoeppeler) [reply module/generic/SUT_generic.py]: Thanks, I found it definitely useful because I needed the same configurations/variables in the teardown  script.

**Pull request merged**